lib/HunkView.ml

```diff
@@ -1,9 +1,15 @@
 open Nottui
 module W = Nottui_widgets
 
+(* Constants and Formatting Functions *)
+let added_marker (content : Ui.t) : Ui.t = Ui.hcat [ W.string "+"; content ]
+let removed_marker (content : Ui.t) : Ui.t = Ui.hcat [ W.string "-"; content ]
+let unchanged_marker (content : Ui.t) : Ui.t = Ui.hcat [ W.string ""; content ]
```

Comment:
Is this any different than just returning `content` ?

Reply:
Why did you remove all the functions ?  These seemed useful.

Reply:
Hmm, I thoght that was what you meant by the comment..?

Reply:
Sorry for the confusion, my comment was about `unchanged_marker` only.